website/project/model.py

```diff
@@ -2162,6 +2166,64 @@ def add_tag(self, tag, auth, save=True, log=True):
             if save:
                 self.save()
 
+    def add_citation(self, auth, save=True, log=True, **kwargs):
```

Comment:
Minor: in general, we try to avoid implicit saves; I suggest changing the default for `save` to False.

Note: Saving the `AlternativeCitation` implicitly is OK because it is a necessary side effect. The implicit `self.save` is what I'm referring to.